host/ndc/integration_test.go

```diff
@@ -87,15 +89,34 @@ var (
 	}
 )
 
-func NewNDCIntegrationTestSuite(params NDCIntegrationTestSuiteParams) *NDCIntegrationTestSuite {
```

Comment:
I am not quite sure if we are using this function and the two struct definitions above in our internal repo. If so, I think we need to keep them exported.

Reply:
Revert it to be exported